gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/distinction/DistinctByIsomorphismTestBase.java

```diff
@@ -0,0 +1,40 @@
+package org.gradoop.flink.model.impl.operators.distinction;
+
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.impl.GraphCollection;
+
+/**
+ * Created by peet on 09.03.17.
+ */
+public class DistinctByIsomorphismTestBase extends GradoopFlinkTestBase {
```

Comment:
I guess you don't add more sophisticated tests since this is built on top of the already tested CAM representation?